lib/ui/fb_button_template.rb

```diff
@@ -4,6 +4,8 @@ module UI
   ########################### BUTTON TEMPLATE #############################
   # https://developers.facebook.com/docs/messenger-platform/send-api-reference/button-template
   class FBButtonTemplate < UI::BaseUiElement
+    include UI::Common::HasButtons
```

Comment:
Forgot to add it in the last version - it'd be nice to release `v0.2.1` with this fix. I'll post a separate PR.

Reply:
Nice, thanks